packages/home-spa/src/mocks/stub.js

```diff
@@ -109,4 +109,47 @@ export const microserviceMock = [
       ],
       illustration: 'img/User-profile-illustration.svg',
    },
-];
\ No newline at end of file
+];
+
+export const teamMembers = [
```

Comment:
Can you get this data from the Rover Group? `one-portal-devel`.
And even the profile pictures.

Reply:
@deshmukhmayur haven't we already discussed fetching rover information after the MVP @deshmukhmayur @npatil9 @gisumit

Reply:
Oh yes we did. I forgot about it.
Nevermind my comment.